docs/sections/user_guide/api/config.rst

```diff
@@ -2,4 +2,5 @@
 ======================
 
 .. automodule:: uwtools.api.config
+   :inherited-members: UserDict
```

Comment:
This tells Sphinx to produce documentation for methods inherited from parent classes, but to stop at and not report on methods inherited from `UserDict`. So, for example, the docs for `YAMLConfig` show usage for `derefernce()`, inherited from `Config` but do not delve deeper into the inheritance hierarchy as e.g. docs for `UserDict` are available [here](https://docs.python.org/3/library/collections.html#collections.UserDict).